crates/trie/common/src/proofs.rs

```diff
@@ -136,6 +136,120 @@ impl MultiProof {
     }
 }
 
+/// The state multiproof of target accounts and multiproofs of their storage tries.
+/// Multiproof is effectively a state subtrie that only contains the nodes
+/// in the paths of target accounts.
```

Comment:
can we clarify the difference with normal `MultiProof`?

Reply:
yep will do